src/tree/FunctionAppProvider.ts

```diff
@@ -121,3 +116,59 @@ async function getDefaultRuntime(actionContext: IActionContext): Promise<Project
 
     return <ProjectRuntime>runtime;
 }
+
+async function createFunctionAppSettings(context: IAppSettingsContext, projectRuntime: ProjectRuntime, projectLanguage: string | undefined): Promise<NameValuePair[]> {
+    const appSettings: NameValuePair[] = [];
+
+    const cliFeedAppSettings: { [key: string]: string } = await getCliFeedAppSettings(projectRuntime);
+    for (const key of Object.keys(cliFeedAppSettings)) {
+        appSettings.push({
+            name: key,
+            value: cliFeedAppSettings[key]
+        });
+    }
+
+    appSettings.push({
+        name: 'AzureWebJobsStorage',
+        value: context.storageConnectionString
+    });
+
+    // This setting only applies for v1 https://github.com/Microsoft/vscode-azurefunctions/issues/640
+    if (projectRuntime === ProjectRuntime.v1) {
+        appSettings.push({
+            name: 'AzureWebJobsDashboard',
+            value: context.storageConnectionString
+        });
+    }
+
+    // These settings only apply for Windows https://github.com/Microsoft/vscode-azurefunctions/issues/625
+    if (context.os === 'windows') {
+        appSettings.push({
+            name: 'WEBSITE_CONTENTAZUREFILECONNECTIONSTRING',
```

Comment:
Huh, maybe time to make a constant for these.  I can think of somewhere else that they're used 👼

Reply:
Lol yes (after the release)